lib/analytical/modules/google_tag_manager.rb

```diff
@@ -28,19 +28,20 @@ def init_javascript(location)
       def event(*args) # name, options, callback
         <<-JS.gsub(/^ {10}/, '')
           var gtmVariables = {};
-          
+
           gtmVariables.event = "Integration GA-GTM Event";
           gtmVariables['eventCategory'] = options['eventCategory'];
           gtmVariables['eventAction'] = options['eventAction'];
           gtmVariables['eventValue'] = options['eventValue'];
-          
+
           try {
             var eventLabels = JSON.parse(options['eventLabel']);
+            gtmVariables['eventValue'] = options['eventValue'];
```

Comment:
isn't this supposed to be "eventLabel" here @brizido ?

Reply:
```suggestion
            gtmVariables['eventLabel'] = options['eventLabel'];
```
Yes. I think it is what Mariana is saying.